components/ImageSlider.js

```diff
@@ -1,112 +1,68 @@
-import styled from "styled-components";
-import { images } from "../lib/shoes";
-import shoes from "../lib/shoes";
 import { useState } from "react";
+import styled from "styled-components";
+import { FaArrowCircleRight, FaArrowCircleLeft } from "react-icons/fa";
+
+export default function ImageSlider({ pictures }) {
+  const [index, setIndex] = useState(0);
 
-const pics = [
-  {
-    id: 1,
-    img1: [
-      "https://res.cloudinary.com/dvze0tvsd/image/upload/v1670928242/capstone-project/Nike/Off-White%20X%20Nike%20Air%20Terra%20Forma%20Wheat/IMG_4260_wtziea.png",
-      "https://res.cloudinary.com/dvze0tvsd/image/upload/v1670928242/capstone-project/Nike/Off-White%20X%20Nike%20Air%20Terra%20Forma%20Wheat/IMG_4262_w4yldp.png",
-      "https://res.cloudinary.com/dvze0tvsd/image/upload/v1670928245/capstone-project/Nike/Off-White%20X%20Nike%20Air%20Terra%20Forma%20Wheat/IMG_4261_zkfbaw.png",
-      "https://res.cloudinary.com/dvze0tvsd/image/upload/v1670928243/capstone-project/Nike/Off-White%20X%20Nike%20Air%20Terra%20Forma%20Wheat/IMG_4264_zqafzm.png",
-    ],
-  },
-];
+  const allPictures = pictures;
 
-export default function ImageSlider() {
+  function handleNext() {
+    if (index < allPictures.length) {
+      setIndex((index + 1) % allPictures.length);
+    }
+  }
+  function handlePrev() {
+    if (index > 0) {
+      setIndex((index - 1) % allPictures.length);
+    }
+  }
   return (
     <>
       <StyledSlider className="slider">
-        <div className="slides">
-          <input type="radio" name="radio-btn" id="radio1" />
-          <input type="radio" name="radio-btn" id="radio2" />
-          <input type="radio" name="radio-btn" id="radio3" />
-          <input type="radio" name="radio-btn" id="radio4" />
-
-          <div className="slide first">
-            <img src="/1.PNG" />
-          </div>
-          <div className="slide">
-            <img src="/2.PNG" />
-          </div>
-        </div>
-        <div className="navigation-manual">
-          <label htmlFor="radio1" className="manual-btn"></label>
-          <label htmlFor="radio2" className="manual-btn"></label>
-          <label htmlFor="radio3" className="manual-btn"></label>
-          <label htmlFor="radio4" className="manual-btn"></label>
+        <div className="slide">
+          <img src={allPictures[index]} />
         </div>
+        <StyledIconContainer>
+          <FaArrowCircleLeft
+            size="25px"
+            className="left"
```

Comment:
Combining styled components with css classes is considered bad practice. You could create a Styled component like this:
const ArrowLeft = styled(FaArrowCircleLeft)``
and apply the same styling as in your class :)